app/models/setting.py

```diff
@@ -356,3 +375,12 @@ def is_omise_activated(self):
             return True
         else:
             return False
+
+    @hybrid_property
+    def is_paytm_activated(self):
```

Comment:
Why didn't column you add a boolean for this?

Reply:
@kushthedude Because it's a violation of [Third Normal Form](https://en.wikipedia.org/wiki/Third_normal_form).

Reply:
No, it isn't. What if I want to add PayTM API keys but not activate PayTM. Or I have added the keys and everything, but I want to disable PayTM. Now I can't do that unless I remove one of the keys, which seems unnecessary

Reply:
@iamareebjamal Yaa, it won't be in that case.

Reply:
@mrsaicharan1 I think you missed it

Reply:
Okay, I will make it a boolean field.